docs/source/modules/convection_simple_betts_miller.rst

```diff
@@ -0,0 +1,93 @@
+Convection scheme: Simple Betts-Miller
+=================================
+
+
+Summary
+-------
+The simplified Betts-Miller (SBM) scheme is a moist adjustment convection scheme developed by [Frierson2007]_ and the motifications described in [OGormanSchneider2008]_. The SBM scheme is Isca's default convection scheme. The key purpose of the routine is output a moisture tendency ``deltaq`` and a temperature tendency ``deltaT``. 
+
+
+Logic of the routine
+-------
+
+1. Perform a particle ascent to calculating the environmental vertical profile (moisture and temperature) and convective available potential energy (CAPE) within a grid cell.
+
+2. Test if the convection scheme should trigger. The trigger is: :math:`CAPE>0`.
+
+3. Calculate the temperature (:math:`T_{ref}`) and moisture reference profiles (:math:`q_{ref}`). Think of these profiles as the desirable parcel properties for the atmosphere to be stable (with respect to convection).
+
+4. Calculate the first guess reference profiles using equations 1-2 from [Frierson2007]_: 
+
+:math:`\partial q = - \frac{q-q_{ref}}{\tau}`,
+
+:math:`\partial T = - \frac{T-T_{ref}}{\tau}`.
+
+These profiles are called *first guess* profiles as they will later be corrected (to conserve enthalpy).
+
+5. Determine if deep convection, shallow convection or no convection occurs in the grid cell (see Section 2 of [Frierson2007]_ for how these are seperated). If the parcel profile is more unstable than the reference profile then it will generate rain (this will occur for both the moisture and temperature profiles to account that condensation can occur due to an increase in moisture or a decrease in temperature).
```

Comment:
> more unstable than the reference profile 
I'd say "unstable with respect to the reference profile"

And then instead of 
> it will generate rain

"The scheme will perform the adjustment and generate rain"? Or something like that...

Reply:
I rewrote this section a bit. Both suggested incorporated.